internal/dataplane/parser/translators/httproute_test.go

```diff
@@ -153,13 +154,37 @@ func TestGeneratePluginsFromHTTPRouteFilters(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "extension-refs",
```

Comment:
I'd be inclined to be more descriptive in those subtest names. Would you be able to provide more details in it about the added subtest? Is it possible to also add a case which would fail (as expected) and assert on the failure?

Reply:
I've changed the function signature to return an error in case the plugin is not supported by Kong and added a test case for it 👍